doc/Administration-guide/System-installation/index.md

```diff
@@ -5,39 +5,52 @@ permalink: /:path/
 nav: /:path/Administration-guide/System-installation/
 sort_idx: 20
 ---
-* [Introduction](#introduction)
-* [System requirements](#system-requirements)
-* [Supported OS](#supported-os)
-* [Third party components](#third-party-components)
-  * [Zookeeper](#zookeeper)
-  * [SQL database](#sql-database)
-  * [NoSQL database](#nosql-database)
-* [Installing Kaa](#installing-kaa)
-  * [Single node installation](#single-node-installation)
-  * [Node cluster setup](#node-cluster-setup)
-  * [AWS deployment](#aws-deployment)
-  * [Planning your deployment](#planning-your-deployment)
-* [Troubleshooting](#troubleshooting)
+
+* TOC
+{:toc}
 
 ## Introduction
 
 This guide overview over kaa platform installation.
```

Comment:
Is it better to say: "This guide provides an overview of Kaa platform installation."?